dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/AuthorizationCodeHandler.java

```diff
@@ -143,18 +145,18 @@ ProviderAuthenticationResult sendTokenRequestAndValidateResult(OidcClientRequest
             String message = Tr.formatMessage(tc, "OIDC_CLIENT_NULL_TOKEN_ENDPOINT", clientId);
             throw new MalformedURLException(message);
         }
-        HashMap<String, String> tokens = oidcClientUtil.getTokensFromAuthzCode(url,
-                clientId,
-                clientConfig.getClientSecret(),
-                redirectUrl,
-                authzCode,
-                clientConfig.getGrantType(),
-                sslSocketFactory,
-                clientConfig.isHostNameVerificationEnabled(),
-                clientConfig.getTokenEndpointAuthMethod(),
-                OIDCClientAuthenticatorUtil.getResources(clientConfig),
-                clientConfig.getTokenRequestParams(),
-                clientConfig.getUseSystemPropertiesForHttpClientConnections());
+        Builder tokenRequestBuilder = new TokenRequestor.Builder(url, clientId, clientConfig.getClientSecret(), redirectUrl, authzCode);
+        tokenRequestBuilder.sslSocketFactory(sslSocketFactory);
+        tokenRequestBuilder.grantType(clientConfig.getGrantType());
```

Comment:
should we modify the grant type (hard coded to `authorization_code` currently) using what's in the config?

from the spec:
```
The call to the token endpoint must include the following parameters (as specified by the OpenID Connect specification):
...
The grant_type value set to the constant authorization_code
```

Reply:
nvm; this piece is for oidc and not jakarta